eng/PatchConfig.props

```diff
@@ -27,9 +27,10 @@ Later on, this will be checked using this condition:
   </PropertyGroup>
   <PropertyGroup Condition=" '$(VersionPrefix)' == '2.2.2' ">
     <PackagesInPatch>
-      @aspnet/signalr;
```

Comment:
FYI @BrennanConroy  

Intentional removal? We need this if we want to patch NPM packages.

Reply:
😢

Reply:
Bad merge.

Reply:
@pakrym You're fixing it?